### ISSUE TITLE
MySQL driver is broken

### ISSUE THREAD

**Issue by unrar** [unrar/midb#1]
When trying to serve an API for a MySQL database, we get this nice error:

```
server_model.rb:287:in `block (2 levels) in get_all_entries': undefined method `length' for #<Mysql2::Result:0x000000024748c8> (NoMethodError)
```

This is because the server model doesn't discriminate between database engines, and turns out a `Mysql2::Result` doesn't have a length attribute.

**Comment by unrar** [comment]
(**FIX**) Add a `length` method to the DbEngine model so to discriminate between engines.

**Comment by unrar** [comment]
**Fixed** in v1.0.1!
